src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -300,6 +302,11 @@ export function checkEligibility(location, siteId) {
       type: FORM_ELIGIBILITY_CHECKS,
     });
 
+    setTimeout(
+      () => dispatch({ type: FORM_SHOW_ELIGIBILITY_LOADING_OVERLAY }),
+      3000,
```

Comment:
I feel like 3 seconds is a long time to wait for this. It also doesn't necessarily help with flickering, since I don't think we have any evidence that if a request takes 3 seconds, it takes significantly longer than 3 seconds.

Reply:
This requirement was suggested by @outerpress.  We could adjust this or remove the delay entirely.